aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1732,9 +1738,9 @@ def _not_found(self, method: str) -> ResponseBuilder:
 
         handler = self._lookup_exception_handler(NotFoundError)
         if handler:
-            return ResponseBuilder(handler(NotFoundError()))
+            return self.response_builder_class(handler(NotFoundError()))
```

Comment:
same as before - is there a reason why we are allocating extra memory to have `ResponseBuilder` into a public attribute?